sui/Cargo.toml

```diff
@@ -23,6 +23,7 @@ num_cpus = "1.13.1"
 base64ct = { version = "1.5.0", features = ["alloc"] }
 ed25519-dalek = { version = "1.0.1", features = ["batch", "serde"] }
 rocksdb = "0.18.0"
+parking_lot = "0.12.0"
```

Comment:
This seems to be the result of a merge error.

Reply:
You are right!